planner/core/find_best_task.go

```diff
@@ -107,6 +107,8 @@ func (p *baseLogicalPlan) findBestTask(prop *property.PhysicalProperty) (bestTas
 		// Next, get the bestTask with enforced prop
 		prop.Items = []property.Item{}
 	}
+	// For props passed down from caller, we need to combine it with the a certain physical plan's props
+	// and compute what props needed to pass to child.
```

Comment:
This comment is irrelevant to the function call `exhaustPhysicalPlans()`, I think it's not needed. Or can you refine the comment of the function `exhaustPhysicalPlans()` in the interface definition of `LogicalPlan`, which is located in "planner/core/plan.go"